flow-decomposition/src/main/java/com/powsybl/flow_decomposition/XnecSelector5percPtdf.java

```diff
@@ -17,6 +17,8 @@
 import java.util.stream.Collectors;
 
 /**
+ * This class will select branches based on the 5% zonal PTDF criteria.
```

Comment:
*OR* if the branch is an interconnection (it has 2 functions)

Reply:
done